Sources/Gtk/Widgets/ModifierBox.swift

```diff
@@ -0,0 +1,58 @@
+import CGtk
+
+/// Helper Box that uses GtkBox under the hood but only allows one child.
+open class ModifierBox: Widget {
+    public private(set) var child: Widget?
+
+    public override init() {
+        super.init()
+        widgetPointer = gtk_box_new(GTK_ORIENTATION_VERTICAL, 0)
```

Comment:
Use `Orientation.vertical.toGtkOrientation()` here (it'll be best for generation)

Reply:
I should probably change this after #37 merged then? Since it will change to `.toGtk()` right?

Reply:
I'll probably merge this first (I might not solve the signal stuff tonight). I'm rebasing the generation branch as I merge your PRs so merge conflicts aren't much of a concern (there aren't many and they're easy to fix with rebasing)